pkg/cmd/roachtest/prometheus.go

```diff
@@ -0,0 +1,166 @@
+// Copyright 2021 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package main
```

Comment:
Curious what you think about putting this in its own package. I would do it without blinking - prometheus is an "addon" that can be installed onto a cluster, and shouldn't need access to any internals.

I realize this isn't easy right now (`Cluster` sits in `main`, `main` needs to import the new package, the new package needs to import main...) but once `Cluster` is in a leaf pkg, it should work.

Not saying this needs to be done now, just please don't refer to anything from `main` in here so we can do it later (I think you're already cognizant of that).